api/steady/serializers/scoresheet_serializer.py

```diff
@@ -11,6 +11,14 @@ class Meta:
         model = ScoreSheet
 
     def create(self, validated_data):
+        """Creates the scoresheet data
+
+        Args:
+            validated_data: Data from entries
```

Comment:
data from the scoresheet that contains entries and those entries contain prompts